### ISSUE TITLE
Error when importing MolModMT in Jupyter-lab

### ISSUE THREAD

**Issue by AngelRuizMoreno** [uibcdf/MolModMT#3]
When I imported MolModMT in jupyter-lab I got the following error:

```
import os
import mdtraj as md
from pdbfixer import PDBFixer
import pdbfixer as pdb
from simtk.openmm.app import PDBFile
import molmodmt as mt

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/openmoltools/amber.py in find_gaff_dat()
    272         try:
--> 273             AMBERHOME = os.path.split(full_path)[0]
    274             AMBERHOME = os.path.join(AMBERHOME, "../")

~/miniconda3/envs/AnalysisMD/lib/python3.6/posixpath.py in split(p)
    106     everything after the final slash.  Either part may be empty."""
--> 107     p = os.fspath(p)
    108     sep = _get_sep(p)

TypeError: expected str, bytes or os.PathLike object, not NoneType

During handling of the above exception, another exception occurred:

ValueError                                Traceback (most recent call last)
<ipython-input-4-db124cf99dc6> in <module>
      4 import pdbfixer as pdb
      5 from simtk.openmm.app import PDBFile
----> 6 import molmodmt as mt

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/molmodmt/__init__.py in <module>
----> 1 from .multitool import *
      2 from . import utils
      3 from . import moldyn as moldyn
      4 from . import molsys as molsys
      5 

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/molmodmt/multitool.py in <module>
     10 
     11 ## Classes
---> 12 from .formats.classes import dict_is_form as _dict_classes_is_form, \
     13     list_forms as _list_classes_forms, \
     14     dict_converter as _dict_classes_converter, \

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/molmodmt/formats/classes/__init__.py in <module>
----> 1 from .base import *

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/molmodmt/formats/classes/base.py in <module>
     18 
     19 for api_form in list_api_forms:
---> 20     module_api_form=_import_module('.'+api_form,base_package)
     21     form_name=module_api_form.form_name
     22     list_forms.append(form_name)

~/miniconda3/envs/AnalysisMD/lib/python3.6/importlib/__init__.py in import_module(name, package)
    124                 break
    125             level += 1
--> 126     return _bootstrap._gcd_import(name[level:], package, level)
    127 
    128 

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/molmodmt/formats/classes/api_yank_Topography.py in <module>
      1 from os.path import basename as _basename
----> 2 from yank import Topography as _yank_Topography
      3 
      4 form_name=_basename(__file__).split('.')[0].replace('api_','').replace('_','.')
      5 

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/yank/__init__.py in <module>
     12 from . import utils
     13 from . import multistate
---> 14 from . import restraints
     15 from . import pipeline
     16 from . import experiment

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/yank/restraints.py in <module>
     35 from simtk import openmm, unit
     36 
---> 37 from . import pipeline
     38 from .utils import methoddispatch, generate_development_feature
     39 

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/yank/pipeline.py in <module>
     29 import numpy as np
     30 import openmmtools as mmtools
---> 31 import openmoltools as moltools
     32 from pdbfixer import PDBFixer
     33 from simtk import openmm, unit

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/openmoltools/__init__.py in <module>
     24 """
     25 
---> 26 from openmoltools import amber_parser, system_checker, utils, packmol, openeye, amber, cirpy, gromacs, schrodinger

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/openmoltools/openeye.py in <module>
      3 import mdtraj as md
      4 from openmoltools.utils import import_, enter_temp_directory, create_ffxml_file
----> 5 from openmoltools.amber import run_antechamber
      6 import logging
      7 

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/openmoltools/amber.py in <module>
    281     return os.path.join(AMBERHOME, 'dat', 'leap', 'parm', 'gaff.dat')
    282 
--> 283 GAFF_DAT_FILENAME = find_gaff_dat()
    284 
    285 

~/miniconda3/envs/AnalysisMD/lib/python3.6/site-packages/openmoltools/amber.py in find_gaff_dat()
    274             AMBERHOME = os.path.join(AMBERHOME, "../")
    275         except:
--> 276             raise(ValueError("Cannot find AMBER GAFF"))
    277 
    278     if AMBERHOME is None:

ValueError: Cannot find AMBER GAFF

```

It looks like is something related to the AMBER HOME path. Should I install Amber to fix the problem. I am managing MolModMT via Conda.

**Comment by dprada** [comment]
Hi @AngelRuizMoreno:
Before checking this error let me specify already that ambertools are required but not Amber. Give me a while to give you a better answer. Thanks!

**Comment by dprada** [comment]
Dear @AngelRuizMoreno:

I can't reproduce the error. And I don't have any variable declared such as 'AMBERHOME' or 'AMBER'. 

Could you please try it with an updated conda env with python 3.7?
Or, please, report here the list of packages you use for me to help you. I guess you have the same problem importing only 'openmoltools', right?

**Comment by AngelRuizMoreno** [comment]
> Dear @AngelRuizMoreno:
> 
> I can't reproduce the error. And I don't have any variable declared such as 'AMBERHOME' or 'AMBER'.
> 
> Could you please try it with an updated conda env with python 3.7?
> Or, please, report here the list of packages you use for me to help you. I guess you have the same problem importing only 'openmoltools', right?

Yes, the problem is inside of 'openmoltools'. I installed ambertools inside of my conda environment as follows:

  conda install ambertools=19 -c ambermd

It does not work. I will upgrade my python version to 3.7 as you advised.

**Comment by dprada** [comment]
I just checked that ambertools is not required. Remove ambertools... I have ambermini, pytraj and openmoltools, openmm and openmoltools. Try to remove it.

**Comment by dprada** [comment]
This is the list of my packages

# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main  
alabaster                 0.7.12                   py37_0  
ambermini                 16.16.0                       7    omnia
asn1crypto                0.24.0                   py37_0  
attrs                     19.1.0                   py37_1  
babel                     2.7.0                      py_0  
backcall                  0.1.0                    py37_0  
biopython                 1.74             py37h7b6447c_0  
blas                      1.0                         mkl  
bleach                    3.1.0                    py37_0  
blosc                     1.16.3               hd408876_0  
bzip2                     1.0.8                h7b6447c_0  
ca-certificates           2019.5.15                     1  
cerberus                  1.1                        py_1    omnia
certifi                   2019.6.16                py37_1  
cffi                      1.12.3           py37h2e261b9_0  
cftime                    1.0.3.4         py37hdd07704_1001  
chardet                   3.0.4                 py37_1003  
clusterutils              0.3.1                    py37_1    omnia
cryptography              2.7              py37h1ba5d50_0  
cudatoolkit               9.2                           0  
curl                      7.65.2               hbc83047_0  
cycler                    0.10.0                   py37_0  
cython                    0.29.12          py37he6710b0_0  
dbus                      1.13.6               h746ee38_0  
decorator                 4.4.0                    py37_1  
defusedxml                0.6.0                      py_0  
docopt                    0.6.2                    py37_0  
docutils                  0.15.1                   py37_0  
entrypoints               0.3                      py37_0  
expat                     2.2.6                he6710b0_0  
fftw3f                    3.3.4                         2    omnia
fontconfig                2.13.0               h9420a91_0  
freetype                  2.9.1                h8a8886c_1  
glib                      2.56.2               hd408876_0  
gmp                       6.1.2                h6c8ec71_1  
griddataformats           0.5.0                      py_0    conda-forge
gsd                       1.8.0            py37hc1659b7_1    conda-forge
gst-plugins-base          1.14.0               hbbd80ab_1  
gstreamer                 1.14.0               hb453b48_1  
hdf4                      4.2.13               h3ca952b_2  
hdf5                      1.10.4               hb1b8bf9_0  
hdf5-1820                 1.8.20                        0    omnia
icu                       58.2                 h9c2bf20_1  
idna                      2.8                      py37_0  
imagesize                 1.1.0                    py37_0  
intel-openmp              2019.4                      243  
ipykernel                 5.1.1            py37h39e3cac_0  
ipython                   7.7.0            py37h39e3cac_0  
ipython_genutils          0.2.0                    py37_0  
ipywidgets                7.5.1                      py_0  
jedi                      0.13.3                   py37_0  
jinja2                    2.10.1                   py37_0  
joblib                    0.13.2                   py37_0  
jpeg                      9c                h14c3975_1001    conda-forge
json5                     0.8.5                      py_0  
jsonschema                3.0.1                    py37_0  
jupyter                   1.0.0                    py37_7  
jupyter_client            5.3.1                      py_0  
jupyter_console           6.0.0                    py37_0  
jupyter_core              4.5.0                      py_0  
jupyterlab                1.0.2            py37hf63ae98_0  
jupyterlab_server         1.0.0                      py_1  
kiwisolver                1.1.0            py37he6710b0_0  
krb5                      1.16.1               h173b8e3_7  
latexcodec                1.0.7                      py_0    conda-forge
libcurl                   7.65.2               h20c2e04_0  
libedit                   3.1.20181209         hc058e9b_0  
libffi                    3.2.1                hd88cf55_4  
libgcc                    7.2.0                h69d50b8_2  
libgcc-ng                 9.1.0                hdf63c60_0  
libgfortran-ng            7.3.0                hdf63c60_0  
libnetcdf                 4.6.2             hbdf4f91_1001    conda-forge
libpng                    1.6.37               hbc83047_0  
libsodium                 1.0.16               h1bed415_0  
libssh2                   1.8.2                h1ba5d50_0  
libstdcxx-ng              9.1.0                hdf63c60_0  
libuuid                   1.0.3                h1bed415_2  
libxcb                    1.13                 h1bed415_1  
libxml2                   2.9.9                hea5a465_1  
lz4-c                     1.8.1.2              h14c3975_0  
lzo                       2.10                 h49e0be7_2  
markupsafe                1.1.1            py37h7b6447c_0  
matplotlib                3.1.0            py37h5429711_0  
mdanalysis                0.19.2          py37hf484d3e_1000    conda-forge
mdtraj                    1.9.3            py37h00575c5_0    conda-forge
mistune                   0.8.4            py37h7b6447c_0  
mkl                       2019.4                      243  
mkl-service               2.0.2            py37h7b6447c_0  
mkl_fft                   1.0.12           py37ha843d7b_0  
mkl_random                1.0.2            py37hd81dba3_0  
mmtf-python               1.1.2                      py_0    conda-forge
mock                      3.0.5                    py37_0  
modeller                  9.20                     py37_1    omnia
molmodel                  0.0.1                     dev_0    <develop>
molmodmt                  0.0.4                     dev_0    <develop>
mpi4py                    3.0.1            py37hf046da1_0    conda-forge
mpich                     3.2                           4    omnia
mpiplus                   v0.0.1                py37_1000    conda-forge
msgpack-python            0.6.1            py37hfd86e86_1  
nb_conda_kernels          2.2.2                    py37_0  
nbconvert                 5.5.0                      py_0  
nbformat                  4.4.0                    py37_0  
nbsphinx                  0.4.2                      py_0    conda-forge
ncurses                   6.1                  he6710b0_1  
netcdf4                   1.4.2            py37h808af73_0  
networkx                  2.3                        py_0  
nglview                   2.7.0              pyh5ca1d4c_0    conda-forge
nodejs                    10.13.0              he6710b0_0  
notebook                  6.0.0                    py37_0  
numexpr                   2.6.9            py37h9e4a6bb_0  
numpy                     1.16.4           py37h7e9f1db_0  
numpy-base                1.16.4           py37hde5b4d6_0  
numpydoc                  0.9.1                      py_0  
openmm                    7.3.1           py37_cuda92_rc_2    omnia
openmmtools               0.18.3                   py37_0    omnia
openmoltools              0.8.3                    py37_1    omnia
openssl                   1.1.1c               h7b6447c_1  
oset                      0.1.3                    py37_1    omnia
packaging                 19.0                     py37_0  
pandas                    0.25.0           py37he6710b0_0  
pandoc                    2.2.3.2                       0  
pandocfilters             1.4.2                    py37_1  
parmed                    3.2.0                    py37_0    omnia
parso                     0.5.0                      py_0  
patsy                     0.5.1                    py37_0  
pcre                      8.43                 he6710b0_0  
pdbfixer                  1.5                      py37_0    omnia
pexpect                   4.7.0                    py37_0  
pickleshare               0.7.5                    py37_0  
pip                       19.1.1                   py37_0  
prometheus_client         0.7.1                      py_0  
prompt_toolkit            2.0.9                    py37_0  
ptyprocess                0.6.0                    py37_0  
pybtex                    0.22.2                   py37_0    conda-forge
pybtex-docutils           0.2.1                    py37_1    omnia
pycparser                 2.19                     py37_0  
pygments                  2.4.2                      py_0  
pymbar                    3.0.3            py37h3010b51_3    conda-forge
pyopenssl                 19.0.0                   py37_0  
pyparsing                 2.4.0                      py_0  
pyqt                      5.9.2            py37h05f1152_2  
pyrsistent                0.14.11          py37h7b6447c_0  
pysocks                   1.7.0                    py37_0  
pytables                  3.5.2            py37h71ec239_1  
python                    3.7.4                h265db76_1  
python-dateutil           2.8.0                    py37_0  
pytraj                    2.0.4                    py37_0    ambermd
pytz                      2019.1                     py_0  
pyyaml                    5.1.1            py37h7b6447c_0  
pyzmq                     18.0.0           py37he6710b0_0  
qt                        5.9.7                h5867ecd_1  
qtconsole                 4.5.2                      py_0  
readline                  7.0                  h7b6447c_5  
requests                  2.22.0                   py37_0  
scikit-learn              0.21.2           py37hd81dba3_0  
scipy                     1.3.0            py37h7c811a0_0  
seaborn                   0.9.0                    py37_0  
send2trash                1.5.0                    py37_0  
setuptools                41.0.1                   py37_0  
sip                       4.19.8           py37hf484d3e_0  
six                       1.12.0                   py37_0  
snappy                    1.1.7                hbae5bb6_3  
snowballstemmer           1.9.0                      py_0  
sphinx                    2.1.2                      py_0  
sphinx_rtd_theme          0.4.3                      py_0  
sphinxcontrib-applehelp   1.0.1                      py_0  
sphinxcontrib-bibtex      0.4.2                      py_0    conda-forge
sphinxcontrib-devhelp     1.0.1                      py_0  
sphinxcontrib-htmlhelp    1.0.2                      py_0  
sphinxcontrib-jsmath      1.0.1                      py_0  
sphinxcontrib-qthelp      1.0.2                      py_0  
sphinxcontrib-serializinghtml 1.1.3                      py_0  
sqlite                    3.29.0               h7b6447c_0  
statsmodels               0.10.1           py37hdd07704_0  
terminado                 0.8.2                    py37_0  
testpath                  0.4.2                    py37_0  
tk                        8.6.8                hbc83047_0  
tornado                   6.0.3            py37h7b6447c_0  
tqdm                      4.32.1                     py_0  
traitlets                 4.3.2                    py37_0  
urllib3                   1.24.2                   py37_0  
wcwidth                   0.1.7                    py37_0  
webencodings              0.5.1                    py37_1  
wheel                     0.33.4                   py37_0  
widgetsnbextension        3.5.0                    py37_0  
xz                        5.2.4                h14c3975_4  
yaml                      0.1.7                had09818_2  
yank                      0.24.1                   py37_0    omnia
zeromq                    4.3.1                he6710b0_3  
zlib                      1.2.11               h7b6447c_3  
zstd                      1.3.7                h0b5b093_0

**Comment by hainm** [comment]
Just FYI here is the logic of the code related to AMBERHOME in openmoltools:
https://github.com/choderalab/openmoltools/blob/master/openmoltools/amber.py#L262

**Comment by AngelRuizMoreno** [comment]
> This is the list of my packages
> 
> # Name Version Build Channel
> _libgcc_mutex 0.1 main
> alabaster 0.7.12 py37_0
> ambermini 16.16.0 7 omnia
> asn1crypto 0.24.0 py37_0
> attrs 19.1.0 py37_1
> babel 2.7.0 py_0
> backcall 0.1.0 py37_0
> biopython 1.74 py37h7b6447c_0
> blas 1.0 mkl
> bleach 3.1.0 py37_0
> blosc 1.16.3 hd408876_0
> bzip2 1.0.8 h7b6447c_0
> ca-certificates 2019.5.15 1
> cerberus 1.1 py_1 omnia
> certifi 2019.6.16 py37_1
> cffi 1.12.3 py37h2e261b9_0
> cftime 1.0.3.4 py37hdd07704_1001
> chardet 3.0.4 py37_1003
> clusterutils 0.3.1 py37_1 omnia
> cryptography 2.7 py37h1ba5d50_0
> cudatoolkit 9.2 0
> curl 7.65.2 hbc83047_0
> cycler 0.10.0 py37_0
> cython 0.29.12 py37he6710b0_0
> dbus 1.13.6 h746ee38_0
> decorator 4.4.0 py37_1
> defusedxml 0.6.0 py_0
> docopt 0.6.2 py37_0
> docutils 0.15.1 py37_0
> entrypoints 0.3 py37_0
> expat 2.2.6 he6710b0_0
> fftw3f 3.3.4 2 omnia
> fontconfig 2.13.0 h9420a91_0
> freetype 2.9.1 h8a8886c_1
> glib 2.56.2 hd408876_0
> gmp 6.1.2 h6c8ec71_1
> griddataformats 0.5.0 py_0 conda-forge
> gsd 1.8.0 py37hc1659b7_1 conda-forge
> gst-plugins-base 1.14.0 hbbd80ab_1
> gstreamer 1.14.0 hb453b48_1
> hdf4 4.2.13 h3ca952b_2
> hdf5 1.10.4 hb1b8bf9_0
> hdf5-1820 1.8.20 0 omnia
> icu 58.2 h9c2bf20_1
> idna 2.8 py37_0
> imagesize 1.1.0 py37_0
> intel-openmp 2019.4 243
> ipykernel 5.1.1 py37h39e3cac_0
> ipython 7.7.0 py37h39e3cac_0
> ipython_genutils 0.2.0 py37_0
> ipywidgets 7.5.1 py_0
> jedi 0.13.3 py37_0
> jinja2 2.10.1 py37_0
> joblib 0.13.2 py37_0
> jpeg 9c h14c3975_1001 conda-forge
> json5 0.8.5 py_0
> jsonschema 3.0.1 py37_0
> jupyter 1.0.0 py37_7
> jupyter_client 5.3.1 py_0
> jupyter_console 6.0.0 py37_0
> jupyter_core 4.5.0 py_0
> jupyterlab 1.0.2 py37hf63ae98_0
> jupyterlab_server 1.0.0 py_1
> kiwisolver 1.1.0 py37he6710b0_0
> krb5 1.16.1 h173b8e3_7
> latexcodec 1.0.7 py_0 conda-forge
> libcurl 7.65.2 h20c2e04_0
> libedit 3.1.20181209 hc058e9b_0
> libffi 3.2.1 hd88cf55_4
> libgcc 7.2.0 h69d50b8_2
> libgcc-ng 9.1.0 hdf63c60_0
> libgfortran-ng 7.3.0 hdf63c60_0
> libnetcdf 4.6.2 hbdf4f91_1001 conda-forge
> libpng 1.6.37 hbc83047_0
> libsodium 1.0.16 h1bed415_0
> libssh2 1.8.2 h1ba5d50_0
> libstdcxx-ng 9.1.0 hdf63c60_0
> libuuid 1.0.3 h1bed415_2
> libxcb 1.13 h1bed415_1
> libxml2 2.9.9 hea5a465_1
> lz4-c 1.8.1.2 h14c3975_0
> lzo 2.10 h49e0be7_2
> markupsafe 1.1.1 py37h7b6447c_0
> matplotlib 3.1.0 py37h5429711_0
> mdanalysis 0.19.2 py37hf484d3e_1000 conda-forge
> mdtraj 1.9.3 py37h00575c5_0 conda-forge
> mistune 0.8.4 py37h7b6447c_0
> mkl 2019.4 243
> mkl-service 2.0.2 py37h7b6447c_0
> mkl_fft 1.0.12 py37ha843d7b_0
> mkl_random 1.0.2 py37hd81dba3_0
> mmtf-python 1.1.2 py_0 conda-forge
> mock 3.0.5 py37_0
> modeller 9.20 py37_1 omnia
> molmodel 0.0.1 dev_0 
> molmodmt 0.0.4 dev_0 
> mpi4py 3.0.1 py37hf046da1_0 conda-forge
> mpich 3.2 4 omnia
> mpiplus v0.0.1 py37_1000 conda-forge
> msgpack-python 0.6.1 py37hfd86e86_1
> nb_conda_kernels 2.2.2 py37_0
> nbconvert 5.5.0 py_0
> nbformat 4.4.0 py37_0
> nbsphinx 0.4.2 py_0 conda-forge
> ncurses 6.1 he6710b0_1
> netcdf4 1.4.2 py37h808af73_0
> networkx 2.3 py_0
> nglview 2.7.0 pyh5ca1d4c_0 conda-forge
> nodejs 10.13.0 he6710b0_0
> notebook 6.0.0 py37_0
> numexpr 2.6.9 py37h9e4a6bb_0
> numpy 1.16.4 py37h7e9f1db_0
> numpy-base 1.16.4 py37hde5b4d6_0
> numpydoc 0.9.1 py_0
> openmm 7.3.1 py37_cuda92_rc_2 omnia
> openmmtools 0.18.3 py37_0 omnia
> openmoltools 0.8.3 py37_1 omnia
> openssl 1.1.1c h7b6447c_1
> oset 0.1.3 py37_1 omnia
> packaging 19.0 py37_0
> pandas 0.25.0 py37he6710b0_0
> pandoc 2.2.3.2 0
> pandocfilters 1.4.2 py37_1
> parmed 3.2.0 py37_0 omnia
> parso 0.5.0 py_0
> patsy 0.5.1 py37_0
> pcre 8.43 he6710b0_0
> pdbfixer 1.5 py37_0 omnia
> pexpect 4.7.0 py37_0
> pickleshare 0.7.5 py37_0
> pip 19.1.1 py37_0
> prometheus_client 0.7.1 py_0
> prompt_toolkit 2.0.9 py37_0
> ptyprocess 0.6.0 py37_0
> pybtex 0.22.2 py37_0 conda-forge
> pybtex-docutils 0.2.1 py37_1 omnia
> pycparser 2.19 py37_0
> pygments 2.4.2 py_0
> pymbar 3.0.3 py37h3010b51_3 conda-forge
> pyopenssl 19.0.0 py37_0
> pyparsing 2.4.0 py_0
> pyqt 5.9.2 py37h05f1152_2
> pyrsistent 0.14.11 py37h7b6447c_0
> pysocks 1.7.0 py37_0
> pytables 3.5.2 py37h71ec239_1
> python 3.7.4 h265db76_1
> python-dateutil 2.8.0 py37_0
> pytraj 2.0.4 py37_0 ambermd
> pytz 2019.1 py_0
> pyyaml 5.1.1 py37h7b6447c_0
> pyzmq 18.0.0 py37he6710b0_0
> qt 5.9.7 h5867ecd_1
> qtconsole 4.5.2 py_0
> readline 7.0 h7b6447c_5
> requests 2.22.0 py37_0
> scikit-learn 0.21.2 py37hd81dba3_0
> scipy 1.3.0 py37h7c811a0_0
> seaborn 0.9.0 py37_0
> send2trash 1.5.0 py37_0
> setuptools 41.0.1 py37_0
> sip 4.19.8 py37hf484d3e_0
> six 1.12.0 py37_0
> snappy 1.1.7 hbae5bb6_3
> snowballstemmer 1.9.0 py_0
> sphinx 2.1.2 py_0
> sphinx_rtd_theme 0.4.3 py_0
> sphinxcontrib-applehelp 1.0.1 py_0
> sphinxcontrib-bibtex 0.4.2 py_0 conda-forge
> sphinxcontrib-devhelp 1.0.1 py_0
> sphinxcontrib-htmlhelp 1.0.2 py_0
> sphinxcontrib-jsmath 1.0.1 py_0
> sphinxcontrib-qthelp 1.0.2 py_0
> sphinxcontrib-serializinghtml 1.1.3 py_0
> sqlite 3.29.0 h7b6447c_0
> statsmodels 0.10.1 py37hdd07704_0
> terminado 0.8.2 py37_0
> testpath 0.4.2 py37_0
> tk 8.6.8 hbc83047_0
> tornado 6.0.3 py37h7b6447c_0
> tqdm 4.32.1 py_0
> traitlets 4.3.2 py37_0
> urllib3 1.24.2 py37_0
> wcwidth 0.1.7 py37_0
> webencodings 0.5.1 py37_1
> wheel 0.33.4 py37_0
> widgetsnbextension 3.5.0 py37_0
> xz 5.2.4 h14c3975_4
> yaml 0.1.7 had09818_2
> yank 0.24.1 py37_0 omnia
> zeromq 4.3.1 he6710b0_3
> zlib 1.2.11 h7b6447c_3
> zstd 1.3.7 h0b5b093_0

I create a clean conda environment using python 3.7.4. Same version as you.
Then I try to install  MolModMT  using the conda package (conda install -c uibcdf molmodmt) and I got the following error:

```
UnsatisfiableError: The following specifications were found to be incompatible with each other:



Package ncurses conflicts for:
python=3.7.4 -> ncurses[version='>=6.1,<7.0a0']
Package parmed conflicts for:
molmodmt -> parmed
Package nglview conflicts for:
molmodmt -> nglview
Package pip conflicts for:
python=3.7.4 -> pip
Package zlib conflicts for:
python=3.7.4 -> zlib[version='>=1.2.11,<1.3.0a0']
Package tk conflicts for:
python=3.7.4 -> tk[version='>=8.6.8,<8.7.0a0']
Package pytraj conflicts for:
molmodmt -> pytraj
Package numpy conflicts for:
molmodmt -> numpy
Package libffi conflicts for:
python=3.7.4 -> libffi[version='>=3.2.1,<4.0a0']
Package yank conflicts for:
molmodmt -> yank
Package libstdcxx-ng conflicts for:
python=3.7.4 -> libstdcxx-ng[version='>=7.3.0']
Package openmm conflicts for:
molmodmt -> openmm
Package pdbfixer conflicts for:
molmodmt -> pdbfixer
Package mdtraj conflicts for:
molmodmt -> mdtraj
Package jupyterlab conflicts for:
molmodmt -> jupyterlab
Package molmodel conflicts for:
molmodmt -> molmodel
Package sqlite conflicts for:
python=3.7.4 -> sqlite[version='>=3.29.0,<4.0a0']
Package openmmtools conflicts for:
molmodmt -> openmmtools
Package mdanalysis conflicts for:
molmodmt -> mdanalysis
Package xz conflicts for:
python=3.7.4 -> xz[version='>=5.2.4,<6.0a0']
Package readline conflicts for:
python=3.7.4 -> readline[version='>=7.0,<8.0a0']
Package biopython conflicts for:
molmodmt -> biopython
Package libgcc-ng conflicts for:
python=3.7.4 -> libgcc-ng[version='>=7.3.0']
Package openssl conflicts for:
python=3.7.4 -> openssl[version='>=1.1.1c,<1.1.2a']
```
I will try to reproduce your environment specs. I will let you know once I finish. Sorry for the inconveniences.

**Comment by AngelRuizMoreno** [comment]
I found where is the problem.

I just realize that is a missing path when working on Jupyter-lab whit exported python kernels.

I created a main conda env with jupyter-lab (BASE). Then I created another conda envs to manage my packages including MolModMT (I will call MD the conda env with MolModMT). I added the python kernels of MD (among others) to BASE in order to run just one single jupyter-lab containing all my working envs (including MD).

When I work with MolModMT using the MD's python kernel on BASE I got the error we were discussed. Nevertheless, if I launch a jupyter notebook inside of MD environment. There is no error as is shown in the image:

![image](https://user-images.githubusercontent.com/34551143/63462447-989e0080-c420-11e9-951c-b581ac7659da.png)

I found another "error" when setting up the MolModMT dependencies inside of a conda env. The error happens when trying to install Mdtraj from OpenMM channel instead of Conda-forge. Installing Mdtraj from OpenMM channel will show several conflicts whit another libraries (as BioPython, OpenMM, Pytraj, and even with Python 3.7.X, among others). So I strongly advice to install the latest version of Mdtraj from conda-forge.

Adding the OpenMM and Conda-forge channels to the conda env will avoid all the mismatches:

```
$conda config --add channels omnia --add channels conda-forge
$conda install mdtraj
```
I am still working on exporting the AMBERHOME path from my MD env for working on BASE. I will let you know any success about it. Meanwhile, I can use MolModMT, which works nicely!!
